### PR TITLE
Modified my caption - #2

### PR DESCRIPTION
**Pull request by cdcrchrisparker**
Added new info to my slide:
- [x] Introduction to my name
- [ ] Info about me

Associated with issue #2

### CODE REVIEW

File: _slides/05-cdcrchrisparker.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello, my name is **Chris**.
```
Comment (IntellexLLC) [inline _slides/05-cdcrchrisparker.md:6]: Pleasure to meet you **Chris**.  It's nice to meet you. Can you also give us a cool fact about yourself?

File: _slides/05-cdcrchrisparker.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (IntellexLLC) [inline _slides/05-cdcrchrisparker.md:3]: `New code needed`

**Review by IntellexLLC — CHANGES_REQUESTED** [review]

File: _slides/05-cdcrchrisparker.md (hunk 1 above)
Comment (cdcrchrisparker) [reply _slides/05-cdcrchrisparker.md:6]: I'd love to give a cool fact, can't think of anything under the pressure of the moment.

**Review by IntellexLLC — APPROVED** [review]
Nicely down 🚀

**Comment by cdcrchrisparker** [comment]
Rebase and merge here

**Comment by cdcrchrisparker** [comment]
Rebase and merge

**Comment by cdcrchrisparker** [comment]
Rebase and merge

**Pull request merged**